std_msgs/Header header
geometry_msgs/Point[16] swarm_pos